src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/SecurityHelper.cs

```diff
@@ -237,19 +217,7 @@ internal static bool CallerAndAppDomainHaveUnrestrictedWebBrowserPermission()
         /// Safe     - The method denies the caller access to the full exception object.
         /// </SecurityNote>
         [SecuritySafeCritical]
-        internal static bool CallerHasUserInitiatedNavigationPermission()
-        {
-            try
-            {
-                CreateUserInitiatedNavigationPermission();
-                _userInitiatedNavigationPermission.Demand();
-            }
-            catch (SecurityException)
-            {
-                return false;
-            }
-            return true;
-        }
+        internal static bool CallerHasUserInitiatedNavigationPermission() { return true; }
```

Comment:
Only 1 site of use in `MS.Internal.AppModel.AppSecurityManager.SafeLaunchBrowserOnlyIfPossible`. 

This method can be deleted and the call can be removed from the callsite.